Tests/SpeziAccountTests/AccountNotificationsTests.swift

```diff
@@ -49,64 +49,66 @@ private final actor TestStandard: Standard, AccountNotifyConstraint {
     }
 }
 
-
-final class AccountNotificationsTests: XCTestCase {
```

Comment:
Maybe we can keep a simple struct wrapper type `AccountNotificationsTests` doesn't necessarily need to be a Suite. This would then allow you to remove the `// swiftlint:disable:next file_types_order` comment again.